lib/ziti-tunnel-cbs/ziti_tunnel_cbs.c

```diff
@@ -178,24 +169,24 @@ static ssize_t get_app_data_json(char *buf, size_t bufsz, tunneler_io_context io
                 if (tag_value != NULL) {
                     char tag_ref[32];
                     snprintf(tag_ref, sizeof(tag_ref), "$tunneler_id.tag[%s]", tag_name);
-                    string_replace(resolved_source_ip, sizeof(resolved_source_ip), tag_ref, tag_value);
+                    string_replace(source_addr, sizeof(source_addr), tag_ref, tag_value);
                 } else {
                     ZITI_LOG(WARN, "cannot set source_ip='%s': identity %s has no tag named '%s'",
                              source_ip, zid->name, tag_name);
                 }
             }
         }
-        string_replace(resolved_source_ip, sizeof(resolved_source_ip), "$intercepted_port", model_map_get(&app_data_model.data, DST_PORT_KEY));
-        string_replace(resolved_source_ip, sizeof(resolved_source_ip), "$client_ip", model_map_get(&app_data_model.data, SRC_IP_KEY));
-        string_replace(resolved_source_ip, sizeof(resolved_source_ip), "$client_port", model_map_get(&app_data_model.data, SRC_PORT_KEY));
-        model_map_set(&app_data_model.data, SOURCE_IP_KEY, resolved_source_ip);
+        string_replace(source_addr, sizeof(source_addr), "$intercepted_port", app_data.dst_port);
+        string_replace(source_addr, sizeof(source_addr), "$client_ip", app_data.src_ip);
```

Comment:
should we change the `client_` variables to `src_` to match the property names? I'm on the fence, myself.